mm2src/adex_cli/src/scenarios/mm2_proc_mng.rs

```diff
@@ -0,0 +1,174 @@
+use common::log::{error, info};
+use std::path::PathBuf;
+use std::{env, u32};
+use sysinfo::{PidExt, ProcessExt, System, SystemExt};
```

Comment:
These imports are not used, will they be used later?

Reply:
These are actually used in `find_proc_by_name`. If these weren't used I would get a warning )